### ISSUE TITLE
The infinite scroll not working on iPAD?

### ISSUE THREAD

**Issue by ghost** [baobabKoodaa/blog#4]
Hi,

I just browse your site https://www.attejuvonen.fi/ and the front page infinite scroll load on laptop/desktop/mobile phone both (iPhone/android) perfectly but when I try to browse on iPad, the infinite scroll stuck after display three posts on the front page and can't go any further.

Any idea? the iPad I'm using is 2nd generation one kind of old and I don't know if that matter or not.

Thanks!

**Comment by baobabKoodaa** [comment]
Thanks for reporting this! When it gets stuck, is there a loading spinner visible?

**Comment by ghost** [comment]
> Thanks for reporting this! When it gets stuck, is there a loading spinner visible?

Yes, the spinner looks like the gear showing and moving. the iOS on my iPad is 9.0.5 and I use the built-in Safari browser. Also, will the pagination show up when there are too many posts on the front page for the infinite scroll to handle?

**Comment by baobabKoodaa** [comment]
The pagination is supposed to show up in case of any errors, or if JavaScript is disabled (there is no limit to how many posts infinite scroll can handle). Unfortunately I don't have an iPad to debug this with, so I fear I have to leave this unfixed for now.

**Comment by ghost** [comment]
> The pagination is supposed to show up in case of any errors, or if JavaScript is disabled (there is no limit to how many posts infinite scroll can handle). Unfortunately I don't have an iPad to debug this with, so I fear I have to leave this unfixed for now.

no problem at all, mostly is because my Ipad is too old and if no one reported to you for having problems with their IPAD.

Regarding the pagination, that's great and I think people like infinite scroll more especially on the mobile devices, just wonder if that is practical if the blog having a thousand pages and still host as static on Netlify.

I like your theme a lot and the main reason is it load super fast and get 99% speed test score which is impossible to archive if using the WordPress.

Thanks for the GREAT WORK!

**Comment by baobabKoodaa** [comment]
I'm glad to hear you like it :)

The thing with bug reports is that for every 1 person who reports something, there's usually hundreds who experience the bug but don't say anything. So the fact that no one else has reported it doesn't mean much - it could affect a lot of people, or it could affect only very few people.

This implementation of infinite scroll still works fine with a few thousand items. Here's a demo: https://gatsby-starter-infinite-scroll.baobab.fi/

And of course, if you run into any problems, you can just switch to pagination entirely. Pagination is great for almost every use case.
